### ISSUE TITLE
Persist games

### ISSUE THREAD

**Issue by j-toscani** [j-toscani/doppelkopf#39]
**Description (five sentences max.)**
If a game was created, I want it´s state to be persisted as long as a player is associated with it.

**Possible implementation**
Optional and mostly for discussion.

**Acceptance Criteria**
- when a game is created/changed it´s state is persisted server side
- if there are no players associated with the game anymore, the entry is deleted
- Players can join and leave a game

**Out of Scope**
- communication between players
- limiting number of connections to a game

**Document optional improvements** 
Optional. Could lead to a better feature.

depends on #38 #36 #28
